File: pset6/sentimental/mario.py
import cs50

def main():
    while True:
        n = cs50.get_int("Height: ")
        if n > 0 and n <= 8:
            break

    for i in range(n):
        print(" " * (n - 1 - i), end="")            # left triangle
        print("#" * (1 + i), end="")

        print("  ", end="")                         # middle space

        print("#" * (1 + i), end="")                # right triangle
    #   print("" * (n - 1 - i), end="")

        print("")

if __name__ == "__main__":
    main()

File: pset1/Mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int i, j;
    int n;
    do
    {
        n = get_int("Height: ");
    }
        while (n < 1 || n > 8);
    for
    (i = 0; i < n; i++)
    {
        for
        (j = 0; j < n-1-i; ++j)
        printf(" ");                    // left pyramid
        for
        (j = 0; j < 1 + i; ++j)
        printf("#");

        printf("  ");                   // spaces between pyramids
        for
        (j = 0; j < 1 + i; ++j)
        printf("#");
        for
        (j = 0; j < n-1-i; ++j)        // right pyramid
        printf("");

        printf("\n");
    }
}